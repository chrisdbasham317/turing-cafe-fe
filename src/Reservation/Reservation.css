.section--reservation {
  background: #ffffff;
  border: 3px solid #008080;
  border-radius: 13px;
  color: #008080;
  height: 28vh;
  margin: 5px;
  width: 18vw;
}