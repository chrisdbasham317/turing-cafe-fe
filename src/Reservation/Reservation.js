import React from 'react';
import './Reservation.css'



const Reservation = ({id, name, date, time, number, cxReservation}) => {
  return (
    <section data-id={id} className='section section--reservation'>
      <div>
        <h2>{name}</h2>
        <p>{date}</p>
        <p>{time} pm</p>
        <p>Number of Guests: {number}</p>
      </div> 
      <button onClick={event => cxReservation(id, event)}>Cancel</button>
    </section>
  )
}



export default Reservation;