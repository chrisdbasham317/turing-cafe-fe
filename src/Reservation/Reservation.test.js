import React from 'react';
import { shallow } from 'enzyme';
import Reservation from './Reservation';

describe('Reservation', () => {
  let wrapper
  let resData = {
    id: 5,
    name: 'John',
    date: '06/17',
    time: '6:30',
    number: 4
  }
  beforeEach(() => {
    wrapper = shallow(<Reservation
      resData={resData}
    />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})
