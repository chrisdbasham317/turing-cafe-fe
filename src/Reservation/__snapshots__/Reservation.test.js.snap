// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reservation should match the snapshot 1`] = `
<section
  className="section section--reservation"
>
  <div>
    <h2 />
    <p />
    <p />
    <p>
      Number of Guests: 
    </p>
  </div>
  <button
    onClick={[Function]}
  >
    Cancel
  </button>
</section>
`;
